### PR TITLE
Add ScRAT

### PR DESCRIPTION
**Pull request by VladimirShitov**
- Add ScRAT to the list of single-cell transformers. This is a reasonably simple model but might be worth mentioning
- Fix a small typo

### CODE REVIEW

**Comment by VladimirShitov** [comment]
Thank you for your comments; I addressed them :) I'm not sure how to classify input embeddings, though. Would you consider cell embeddings followed by attention layer a "value projection"?

Regarding COMBAT, there is a link to CellXGene.

File: README.md [hunk 1]
```diff
@@ -48,9 +49,10 @@ We welcome contributions to this repository. Please open a pull request or an is
...
+| GPT-4          | 📄(Nature Meth)[W. Hou and Z. Ji 2024](https://www.nature.com/articles/s41592-024-02235-4)                                                                               | [🔍GitHub](https://github.com/Winnie09/GPTCelltype)           | scRNA-seq    | Natural Language, Closed source                                                                                                                                                                                                                                                                                                                            | Ordering: embedding as natural language                                                                | Closed source                                                                   | Closed source                                                       | None (conditional sequence generation, prompting)                      | Cell type annotation                                          |
```
Comment (szalata) [inline README.md:52]: good catch!

File: README.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ We welcome contributions to this repository. Please open a pull request or an is
...
+| ScRAT  | 📄(Bioinformatics)[Mao et al. 2024](https://academic.oup.com/bioinformatics/article/40/2/btae067/7613064)) | [🛠️GitHub](https://github.com/yuzhenmao/ScRAT) | scRNA-seq       | COMBAT: 836k / immune, human ([CELLxGENE](https://cellxgene.cziscience.com/collections/8f126edf-5405-4731-8374-b5ce11f53e82), Haniffa: 528k / immune, human ([CELLxGENE](https://cellxgene.cziscience.com/collections/ddfad306-714d-4cc0-9985-d9072820c530)), SC4: 1.3M / immune, human ([website](http://covid19.cancer-pku.cn/#/summary)). Only evaluated within each dataset | Cells as tokens                                                                | Encoder                                                                         | None                                                    | Phenotype prediction: aggregated per sample cell embeddings are used to predict sample label (e.g., health condition)                                                                  | None                              |
```
Comment (szalata) [inline README.md:13]: as far as I can see, they don't do SSL pre-training, do they? I think the datasets you listed were only used for supervised training, so following the convention it's enough to state "None". 
Aside from that, looks good to me!

**Review by szalata — CHANGES_REQUESTED** [review]
Two small comments to address, aside from that, lgtm!

File: README.md (hunk 2 above)
Comment (VladimirShitov) [reply README.md:13]: They don't, I'll change it. In Supplementary, there is a section on the fine-tuning for transferring predictions from one dataset to another. But this is all on the supervised tasks.

File: README.md [hunk 3]
```diff
@@ -48,9 +49,10 @@ We welcome contributions to this repository. Please open a pull request or an is
...
 | Cell2Sentence  | 📄(ICML)[Levine et al. 2024](https://openreview.net/forum?id=EWt5wsEdvc&referrer=%5Bthe%20profile%20of%20Josue%20Ortega%20Caro%5D(%2Fprofile%3Fid%3D~Josue_Ortega_Caro1)) | [🛠️GitHub](https://github.com/vandijklab/cell2sentence-ft) | scRNA-seq       | Natural Language ([GPT2](https://d4mucfpksywv.cloudfront.net/better-language-models/language_models_are_unsupervised_multitask_learners.pdf)) and [scRNA-seq](https://www.science.org/doi/full/10.1126/science.abl5197?casa_token=KSZInYXxqU4AAAAA%3AuNgeqoX4vxOaMPGAv4UW9_GMy1lMmZ1-QGyx2VBCSbsGWvchKCzdNUvwt-h_yemzugH075TGz6N8fw) (40k / immune, human) | Ordering: embedding as natural language                                                                | Decoder                                                                         | NTP with CE loss                                                    | None                                                                  | Simulation, cell type annotation                              | 
 
+
```
[the remark this answers is not in the document]
Comment (szalata) [reply README.md:55]: not sure what you mean - this line would come after all of the LLMs and as far as I can tell, it's unnecessary.
Thanks for the updates!

[the remark this answers is not in the document]
Comment (VladimirShitov) [reply README.md:55]: Ah, sorry, I thought you meant spaces in the cells. This line is just an artifact

**Comment by VladimirShitov** [comment]
@szalata , could you take a look at the merge conflict? Quite a few lines changed with a recent commit in main. You know the file better, so perhaps it's faster for you to fix it :)

**Comment by szalata** [comment]
great! ofc!

**Pull request merged**